chat-bots/src/Data/Chat/Bot/Monoidal.hs

```diff
@@ -57,24 +59,14 @@ nudgeRight = nudge . Right
 infixr 9 /\
 
 (/\) :: Monad m => Bot m s i o -> Bot m s' i o' -> Bot m (s /\ s') i (o /\ o')
-(/\) (Bot b1) (Bot b2) = Bot $ \(s, s') i -> do
-  (nextState, responses) <- b1 s i
-  (nextState', responses') <- b2 s' i
-  pure $ (,) (nextState, nextState') (responses, responses')
+(/\) b1 = lmap split' . curry Trifunctor.combine b1
```

Comment:
If possible, I think it makes more sense to just use `curry Trifunctor.combine` here and update the type.

The `lmap split'` thing dealing with the same problem we have with the `s` parameter of `(\/)`, and it's not immediately obvious this is the right way to solve it. Or at least, if this is how we solve it, the consistent dual is to require `Monoid s =>` and `rmap (<>)` over `(<|>)`.

Reply:
I think that because we are contravariant on `i` that the semantics are correct here? We split the input and pass it to both bots. That said, there isn't a particular reason we need this particular operation so I'll make it just `curry combine`.

Reply:
Also, on `monoidal-functors` master branch we now have all these combinators available so we can just delete all of them from cofree-bot.